js/src/offcanvas.js

```diff
@@ -109,7 +109,7 @@ class Offcanvas extends BaseComponent {
     this._backdrop.show()
 
     if (!this._config.scroll) {
-      scrollBarHide()
+      new ScrollBarHelper().hide()
```

Comment:
In modal you're setting it in the constructor (`this._scrollBar = new ScrollBarHelper()`) but here you creating a new instance, why?

Reply:
I'm ok either way. I was just confused because for modal you initialize it once and make use of the same instance and in offcanvas you're initialize a new instance everytime you call a method on it.